assets/stylesheets/test-details.scss

```diff
@@ -414,4 +414,14 @@ ul.modcategory {
     .fa-download:hover {
         opacity: inherit;
     }
+}
+
+.test-result-overview-link {
+    top: -0.1em;
+    position: relative;
+    left: 0.2em;
+    opacity: 0.8;
+    font-size:medium;
+    color: #446E9B;
```

Comment:
Don't copy & paste code. SCSS allows to avoid that. However, in this case just get rid of the line as it doesn't make a difference anyways.

Reply:
I understand this comment is outdated because your last comment. Isn't it?

Reply:
I think it looks good now